mne/annotations.py

```diff
@@ -0,0 +1,49 @@
+# Authors: Jaakko Leppakangas <jaeilepp@student.jyu.fi>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+from datetime import datetime
+import time
+
+
+class Annotations(object):
+    """Annotation object for annotating segments of raw data.
+
+    Parameters
+    ----------
+    onset: array of float
+        Annotation time onsets from the beginning of the recording.
+    duration: array of float
+        Durations of the annotations.
+    description: array of str
+        Array of strings containing description for each annotation.
+    orig_time: int | instance of datetime | None
+        Timestamp or a datetime determining the starting time of annotation
+        acquisition. If None (default), starting time is determined from
+        beginning of raw data.
+    """
+
+    def __init__(self, onset, duration, description, orig_time=None):
+
+        if orig_time is not None:
+            if isinstance(orig_time, datetime):
+                orig_time = time.mktime(orig_time.timetuple())
+            elif not np.isscalar(orig_time):
```

Comment:
I think I'd rather have the user do the time adjustment themselves to align with the `raw` time series, rather than us do some adjustment here.